app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -71,113 +71,132 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
+
+      if ($scope.emsCommonModel.default_userid !== '') {
+        $scope.emsCommonModel.default_password = $scope.emsCommonModel.default_verify = miqService.storedPasswordPlaceholder;
+      }
+      if ($scope.emsCommonModel.amqp_userid !== '') {
+        $scope.emsCommonModel.amqp_password = $scope.emsCommonModel.amqp_verify = miqService.storedPasswordPlaceholder;
+      }
+      if ($scope.emsCommonModel.metrics_userid !== '') {
+        $scope.emsCommonModel.metrics_password = $scope.emsCommonModel.metrics_verify = miqService.storedPasswordPlaceholder;
+      }
+      if ($scope.emsCommonModel.ssh_keypair_userid !== '') {
+        $scope.emsCommonModel.ssh_keypair_password = miqService.storedPasswordPlaceholder;
+      }
+      if ($scope.emsCommonModel.bearer_token_exists) {
+        $scope.emsCommonModel.default_userid = "_";
+        $scope.emsCommonModel.default_password = $scope.emsCommonModel.default_verify = miqService.storedPasswordPlaceholder;
+      }
+
+      $scope.afterGet  = true;
+      $scope.modelCopy = angular.copy( $scope.emsCommonModel );
+
+      $scope.populatePostValidationModel();
+
+      miqService.sparkleOff();
+    }
+
+    function getNewEmsFormDataComplete(response, status, headers, config) {
+      var data = response.data;
+
+      $scope.emsCommonModel.zone                            = data.zone;
+      $scope.emsCommonModel.tenant_mapping_enabled          = data.tenant_mapping_enabled;
+      $scope.emsCommonModel.emstype_vm                      = data.emstype_vm;
+      $scope.emsCommonModel.openstack_infra_providers_exist = data.openstack_infra_providers_exist;
+      $scope.emsCommonModel.default_api_port                = '';
+      $scope.emsCommonModel.amqp_api_port                   = '5672';
+      $scope.emsCommonModel.hawkular_api_port               = '443';
+      $scope.emsCommonModel.api_version                     = 'v2';
+      $scope.emsCommonModel.ems_controller                  = data.ems_controller;
+      $scope.emsCommonModel.ems_controller === 'ems_container' ? $scope.emsCommonModel.default_api_port = '8443' : $scope.emsCommonModel.default_api_port = '';
+      $scope.emsCommonModel.default_auth_status             = data.default_auth_status;
+      $scope.emsCommonModel.amqp_auth_status                = data.amqp_auth_status;
+      $scope.emsCommonModel.service_account_auth_status     = data.service_account_auth_status;
+      $scope.emsCommonModel.metrics_auth_status             = true;
+      $scope.emsCommonModel.ssh_keypair_auth_status         = true;
+      $scope.emsCommonModel.hawkular_auth_status            = true;
+      $scope.emsCommonModel.vmware_cloud_api_version        = '9.0';
+      miqService.sparkleOff();
+
+      $scope.afterGet  = true;
+      $scope.modelCopy = angular.copy( $scope.emsCommonModel );
+    }
+
+    function getEmsFormDataFailed(e) {
+      miqService.sparkleOff();
+      console.log(e.message);
```

Comment:
We would always have `e`, although `e.message` could be questionable.

So I'll change it to --
```javascript
if (e.message) {
  console.log(e.message);
}
```